src/main/java/uk/org/tombolo/core/utils/HibernateUtil.java

```diff
@@ -18,6 +25,44 @@ public static void startup() {
         sharedSession = sessionFactory.openSession();
         sharedSession.setCacheMode(CacheMode.NORMAL);
     }
+
+    public static void startUpForPython() {
+        sessionFactory = buildSessionFactoryForPython();
+        sharedSession = sessionFactory.openSession();
+        sharedSession.setCacheMode(CacheMode.NORMAL);
+    }
+
+    private static SessionFactory buildSessionFactoryForPython() {
```

Comment:
Because when called from Python, the whole project is not loaded. Py4j loads the class in jvm and at that point it could access hibernate config file, if it doesn't exist in same path. Thus either i create a hibernate config file in core directory or pass it at run time. Creating another file looked ugly as it would confuse the user which one is main, thus passed the parameters at run time